common/supplemental/plurals.xml

```diff
@@ -155,9 +155,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
         <!-- 4: one,two,many,other -->
```

Comment:
```suggestion
        <!-- 4: one,two,other -->
```

Reply:
After all of the plural rules are done, I'm going to apply the program to normalize the output